### DISCUSSION TITLE
osu!lazer for aarch64

### DISCUSSION THREAD

**Discussion by Smorty100** [ppy/osu#14537] [Ideas]
I know I know, this is really just a convenience thing for the few people who use arm based machines... and it will make the already long list of downlaods in the github even longer. And of coarse it takes a while to compile it for arm BUT it would be really handy if we could just download a for arm compiled lazer version and play directly on our raberry pis or switches running linux.

**Comment by gagahpangeran** [comment]
This needs [osu!framework](https://github.com/ppy/osu-framework) to support linux arm first.

If you have arm device, you are welcome to do it.

You can see old discussion about attempt to compile to linux arm at https://github.com/ppy/osu/issues/7166

**Reply by Smorty100** [reply]
Will I just beable to use the appimage, or will I have to use the source code?

**Reply by gagahpangeran** [reply]
First thing is you have to add all arm native library to osu-framework, make sure it can build and pass all tests, then create pull request for it.

If you are not developer or have limited experience about programming, I think those stuffs is a bit hard for you, and we don't have time and resources to guide beginner to do it.

**Reply by Smorty100** [reply]
Alright... Then this isn't for me I guess. No wonder building Lazer fails all the time.

**Comment by Aurora-Nasa-1** [comment]
i need it
